spec/functional/win32/taskscheduler_spec.rb

```diff
@@ -521,50 +521,113 @@
     context "Service Account Users" do
       let(:service_user) { "LOCAL SERVICE" }
       let(:password) { nil }
-      it "Does not require any password" do
-        expect(@ts.set_account_information(service_user, password)). to be_truthy
-        expect(@ts.account_information.upcase).to eq(service_user)
-      end
-      it "passing account_name will display account_simple_name" do
-        user = 'NT AUTHORITY\LOCAL SERVICE'
-        expect(@ts.set_account_information(user, password)). to be_truthy
-        expect(@ts.account_information.upcase).to eq(service_user)
+      context "when task is Interactive" do
+        let(:interactive) { true }
+        it "Does not require any password" do
+          expect(@ts.set_account_information(service_user, password, interactive)). to be_truthy
```

Comment:
having a space between the . and to, e.g. `. to be_truthy` looks weird to me, but I don't mind if it passes chefstyle.